apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/attach-launch-sidebar.tsx

```diff
@@ -54,6 +55,7 @@ registerPlugin( 'a8c-editor-site-launch', {
 						redirectTo: redirectToWpcomPath,
 						openCheckout,
 						getCurrentLaunchFlowUrl,
+						isInIframe: inIframe(),
```

Comment:
[Changes in 48452](https://github.com/Automattic/wp-calypso/pull/48452/files#diff-20cce9e66c1059231bcccf929f078c7d8f5877a21859175f7795469214ccfb8dR17) introduced this error (the `isInIframe` prop is required by `LaunchContext` but was not provided here), but somehow neither building the site locally, nor the CI checks flagged it.

We should look into having a full automated TypeScript check — pinging @scinos for some guidance (and in case this is related to #49984)

Reply:
We've set `isInIframe` to `false` as default [in the commit where we introduced it](https://github.com/Automattic/wp-calypso/pull/48452/commits/cf11a5a0c42418a3163b68f0c43b95c9d312074e) and the app mounted here is not using it so it's not really a bug. 

I'm not against adding it to have the context consistent across both apps but maybe we should remove the [default](https://github.com/Automattic/wp-calypso/blob/trunk/packages/launch/src/context.ts#L28-L41) to reduce the maintenance effort?
https://kentcdodds.com/blog/how-to-use-react-context-effectively

Reply:
I see why this wasn't a real bug, but it caused a TypeScript error that was never flagged (the only way I came across it was by opening that file in my IDE).

I agree that one way of fixing this is to make the `isInIframe` prop optional — I will go ahead and apply this change in this PR

Reply:
> I agree that one way of fixing this is to make the `isInIframe` prop optional — I will go ahead and apply this change in this PR

It's probably not a good idea to do that since we have it required in Focused Launch 😬 
Let's continue to keep the context using the same interface for both apps.

Reply:
Ok then, I guess we can keep the code in this PR as is.

> maybe we should remove the default value of the `LaunchContext`

This could be an option.

The more pressing issue, for me, is to understand how we can get TypeScript errors like this flagged in our automated build / CI processes